admin/views/school/school-home.ejs

```diff
@@ -38,13 +38,43 @@
             </li>
 
             <li>
+                <% if (pinGenerationEligibilityData.isFamiliarisationPinGenerationAllowed && !pinGenerationEligibilityData.isUnavailableTime) { %>
                 <h2 class="heading-small"><a href="/pupil-pin/generate-familiarisation-pins-overview">Generate pupil PINs - familiarisation check</a></h2>
                 <p>Generate personal identification numbers (PINs) for pupils and view school password for the familiarisation check</p>
+                <% } %>
+                <% if (pinGenerationEligibilityData.isUnavailableTime || !pinGenerationEligibilityData.isFamiliarisationPinGenerationAllowed) { %>
+                <h2 class="heading-small"><a class="font-greyed-out disabled-link" href="#">Generate pupil PINs - familiarisation check</a></h2>
+                <span>Generate personal identification numbers (PINs) for pupils and view school password for the familiarisation check</span>
+                <p>
+                    <strong class="disabled-banner">UNAVAILABLE</strong>
```

Comment:
Does the UI ask for all uppercase?  It's generally considered to be less readable.

Reply:
@jon-shipley the requirement is indeed asking for uppercase.